cardano-node/src/Cardano/Node/Run.hs

```diff
@@ -283,15 +287,16 @@ handleSimpleNode scp runP trace nodeTracers nc onKernel = do
            onKernel nodeKernel
        }
      StdRunNodeArgs
-       { srnBfcMaxConcurrencyBulkSync   = unMaxConcurrencyBulkSync <$> ncMaxConcurrencyBulkSync nc
-       , srnBfcMaxConcurrencyDeadline   = unMaxConcurrencyDeadline <$> ncMaxConcurrencyDeadline nc
-       , srnChainDbValidateOverride     = ncValidateDB nc
-       , srnSnapshotInterval            = ncSnapshotInterval nc
-       , srnDatabasePath                = dbPath
-       , srnDiffusionArguments          = diffusionArguments
-       , srnDiffusionTracers            = diffusionTracers
-       , srnEnableInDevelopmentVersions = ncTestEnableDevelopmentNetworkProtocols nc
-       , srnTraceChainDB                = chainDBTracer nodeTracers
+       { srnBfcMaxConcurrencyBulkSync    = unMaxConcurrencyBulkSync <$> ncMaxConcurrencyBulkSync nc
+       , srnBfcMaxConcurrencyDeadline    = unMaxConcurrencyDeadline <$> ncMaxConcurrencyDeadline nc
+       , srnChainDbValidateOverride      = ncValidateDB nc
+       , srnSnapshotInterval             = ncSnapshotInterval nc
+       , srnDatabasePath                 = dbPath
+       , srnDiffusionArguments           = diffusionArguments
+       , srnDiffusionTracers             = diffusionTracers
+       , srnEnableInDevelopmentVersions  = ncTestEnableDevelopmentNetworkProtocols nc
+       , srnTraceChainDB                 = chainDBTracer nodeTracers
+       , srnMaybeMempoolCapacityOverride = Just (MempoolCapacityBytesOverride (MempoolCapacityBytes 1000))
```

Comment:
Shouldn't this value be coming from the `NodeConfiguration`?

Reply:
Fixed.  Thanks!